shotover-proxy/src/transforms/redis/cache.rs

```diff
@@ -228,7 +228,9 @@ impl SimpleRedisCache {
     /// Clears the cache for the entire table
     /// TODO make this drop only the specified keys not the entire cache
     fn drop_table(&self, _statement: &CassandraStatement) -> Message {
-        Message::from_frame(Frame::Redis(RedisFrame::BulkString("FLUSHDB".into())))
+        Message::from_frame(Frame::Redis(RedisFrame::Array(vec![
```

Comment:
I have no idea why my changes revealed this problem.

Reply:
I moved this into https://github.com/shotover/shotover-proxy/issues/702
It makes sense to address as a follow up because there is already plenty going on in this PR and hand crafting redis requests like we do in the cache transform is something that actual users will almost never do.